integration/e2ecortex/client.go

```diff
@@ -267,6 +334,13 @@ func (c *Client) Query(query string, ts time.Time) (model.Value, error) {
 	return value, err
 }
 
+// QueryExemplars runs an exemplars query
+func (c *Client) QueryExemplars(query string, start, end time.Time) ([]promv1.ExemplarQueryResult, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), c.timeout)
+	defer cancel()
```

Comment:
I suggest we pass ctx from the test itself so we control timeout there

Reply:
It would be better but we cannot directly access the client's timeout in the test.

Reply:
I see nvm it was the same pattern for other APIs. I am good